.github/workflows/release.yml

```diff
@@ -1,6 +1,11 @@
 name: Release
 
-on: { push }
+on:
+  push:
+    branches:
+      - main
```

Comment:
I guess you can adjust the branch filter if you're making changes to it and want to test it.

What branches-ignore filter would you use? I figured it didn't make much sense to run it for all PRs since it basically does the same as the build workflow, but with the addition of signing.

Reply:
nvm, separating release and build makes a lot of sense